src/ui/ResultActions/ResultActionsMenu.ts

```diff
@@ -0,0 +1,126 @@
+import { Component } from '../Base/Component';
+import { ComponentOptions } from '../Base/ComponentOptions';
+import { IResultsComponentBindings } from '../Base/ResultsComponentBindings';
+import { $$ } from '../../utils/Dom';
+import { IQueryResult } from '../../rest/QueryResult';
+import { Assert } from '../../misc/Assert';
+import { Initialization } from '../Base/Initialization';
+import * as _ from 'underscore';
+import { exportGlobally } from '../../GlobalExports';
+import 'styling/_ResultActionsMenu';
+
+export interface IResultActionsMenuOptions {
+  openOnMouseOver?: boolean;
+}
+
+/**
+ * The ResultActionsMenu is a small floating slack-like result actions menu.
```

Comment:
@amoreauCoveo can you help review this  ?

Thank you !

Reply:
```
The _ResultActionsMenu_ component adds a floating result action menu, meant to be used inside result templates (see [Result Templates](https://docs.coveo.com/en/413/javascript-search-framework/result-templates)).

 It is designed to contain other components that can execute actions related to the result, 
 typically the [Quickview]{@link Quickview} and AttachToCase components, available in the Coveo for Salesforce and Coveo for Dynamics integrations.
```

(adaptation de ce qu'on a pour la doc de Salesforce)